include/Pattern.h

```diff
@@ -127,6 +127,7 @@ class EXPORT Pattern : public TrackContentObject
 
 protected slots:
 	void addSteps();
+	void duplicateFirstBarAtEnd();
```

Comment:
Perhaps its only the name I have a problem with... 

If the location of addSteps() is implied and the location of removeSteps() is implied, can't the location of duplicateSteps() be implied too?